src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeValidator.java

```diff
@@ -217,6 +218,39 @@ public static void assertClassificationNewApplicationIfPaymentsArePresent(InputE
         }
     }
 
+    public static void assertPaymentsEnabledForContainerIfPaymentsArePresent(
+        InputEnvelope envelope,
+        boolean paymentsEnabled,
+        List<ContainerMappings.Mapping> mappings
+    ) {
+        if (envelope.payments != null && !envelope.payments.isEmpty()
```

Comment:
you can just use `CollectionUtils.isNotEmpty(listName)`?

Reply:
done